meetings/2016-07-14.md

```diff
@@ -0,0 +1,140 @@
+# Node Foundation TSC Meeting 2016-07-14
+
+## Links
+
+* **Video Recording**: https://www.youtube.com/watch?v=UXlzfJ9zPIY
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/118
+* **Minutes Google Doc**: <https://docs.google.com/document/d/1M29qRvgLU5LedQZ9kpZfWzzlDbNvWB9RolWBcD5ZBco>
+* _Previous Minutes Google Doc: <https://docs.google.com/document/d/1eUpQNVdupAlsyvVVEnu8za6d8aBhOv9bEZK9FVWm8ds>
+
+## Present
+
+* Rod Vagg (TSC)
+* Alexis Campailla (TSC)
+* Jeremiah Senkpiel (TSC)
+* Michael Dawson (Observer)
+* James M Snell (TSC)
+* Bryan Hughes (TSC)
+* Josh Gavant (Observer)
+* William Kapke (Observer)
+
+## Agenda
+
+Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+* Define "Node.js core" - Mark II [#113](https://github.com/nodejs/TSC/issues/113)
+* Regular roll-up reporting for active working groups [#109](https://github.com/nodejs/TSC/issues/109)
+* Define "Node.js core" [#84](https://github.com/nodejs/TSC/issues/84)
+
+## Review of last meeting
+
+* Regular roll-up reporting for active working groups [#109](https://github.com/nodejs/TSC/issues/109)
+* Add @nebrius as a member of the TSC [#108](https://github.com/nodejs/TSC/pull/108)
+* Define "Node.js core" [#84](https://github.com/nodejs/TSC/issues/84)
+
+
+## Standup
+
+
+* Michael Dawson: Made the new “Define Node.js Core” PR
```

Comment:
I'd update this to 
* Michael Dawson: Some interaction and pushed minor changes to "Define Node.js Core" PR